EnglishTranslation-master/QUEST_LV_0100.tsv

```diff
@@ -12321,252 +12321,252 @@ QUEST_LV_0100_20151224_012321	But as you know.. {nl}There are barely enough sold
-QUEST_LV_0100_20151224_012337	I would have tried to do something myself if it was just a few monsters, but something that size..{nl}Just run if you think it's too much for you.
+QUEST_LV_0100_20151224_012335	I was so startled that I ran away... Can't you do something about it?{nl}I'm sure that it'll threaten Orsha if we just leave it there.
+QUEST_LV_0100_20151224_012336	If you really don't mind, please deal with the monster on Bonan Forest Road.{nl}There's nothing good that could come from leaving monsters there anyways.
+QUEST_LV_0100_20151224_012337	I would have tried to do something myself if it was just a few monsters, {nl}but something of that size...{nl}Just run if you think it's too much for you.
 QUEST_LV_0100_20151224_012338	You burned the nest to the ground? {nl}That was a wise decision!{nl}
-QUEST_LV_0100_20151224_012339	The reason monsters kept pouring out from Bonan Forest Road is probably because of that one.{nl}There are even monsters I'm seeing for the first time since the surrounding areas are all forest.
-QUEST_LV_0100_20151224_012340	You've found the diaries of Priest Irma and the Bishop?{nl}Hm.. There are too many parts that I can't get my head around.{nl}
+QUEST_LV_0100_20151224_012339	The reason monsters kept pouring out from Bonan Forest Road is probably because of that one.{nl}There are even monsters I'm seeing for the first time since the surrounding areas are all forests.
+QUEST_LV_0100_20151224_012340	You've found the diaries of Priest Irma and the bishop?{nl}Hm... There are too many parts that I can't get my head around.{nl}
 QUEST_LV_0100_20151224_012341	If they were merely being chased, they could have just run to Orsha.{nl}But they've vanished going through the effort of leaving behind pieces of their diaries.{nl} 
-QUEST_LV_0100_20151224_012342	What I think is.. Maybe they were threatened..{nl}Or they had a mission that they had to complete even though they were being chased?{nl}
-QUEST_LV_0100_20151224_012343	Mission.. I've been looking for traces of Priest Gelija.{nl}I can't make out exactly what it written in the notes left by Gelija either since they are only partially recognizable thanks to the monsters.{nl}
-QUEST_LV_0100_20151224_012344	Priest Pranas said that finding traces of Priest Gelija is very important becuase of the priest's thorough nature.{nl}I also think that we'll be able to narrow down our search radius if we can find all of the clues left by Gelija.{nl}
+QUEST_LV_0100_20151224_012342	It could be... That they were threatened maybe...{nl}Or they had a mission that they had to complete even though they were being chased?{nl}
+QUEST_LV_0100_20151224_012343	A mission... I've been looking for traces of Priest Gelija.{nl}I cannot make out exactly what it written in the notes left by Gelija {nl}either since they are only partially recognizable due to the monsters.{nl}
```

Comment:
i think the sentence/text box would flow better if you put {nl} after "either" instead of before